widgetsnbextension/src/manager.js

```diff
@@ -533,6 +533,141 @@ WidgetManager.prototype._rasterizeEl = function(el) {
 };
 
 /**
+ * Update the widget snapshots for a single cell
+ * @param  {object} cell
+ * @param  {number} index - of the cell
+ * @return {Promise}
+ */
+WidgetManager.prototype._updateCellSnapshots = function(cell, index) {
```

Comment:
@jdfreder this seems to be a rebase issue. All these functions already exist in the file elsewhere.